src/Types/NetworkTypes.ts

```diff
@@ -0,0 +1,20 @@
+export type NetworkGraphNode = {
+  // How far that node is from the root node (ie. how many quorum set hops)
+  // 0 means this is the node being administrated
+  distance: number;
+  // The latest ledger sequence number that this node voted at
+  heard: number;
+  // The identity of the validator
+  node: string;
+  // Quorum set
+  qset: {
+    t: number;
```

Comment:
Could we use more verbose names for `t`and `v` here? I think it'd improve readability / make it easy to understand the code.

Reply:
unfortunately this has to match whats on the server which has some pretty opaque naming.  As we create these fields in local data models i'll definitely be clearer

Reply:
OK, I understand that – I like that you put comments explaining what they mean.